app/jobs/searchgov_domain_destroyer_job.rb

```diff
@@ -1,11 +1,12 @@
 class SearchgovDomainDestroyerJob < ApplicationJob
   queue_as :searchgov
 
-  def perform(searchgov_domain:)
-    searchgov_domain.searchgov_urls.find_each do |url|
-      url.destroy!
-    end
+  def perform(searchgov_domain)
+    searchgov_domain.searchgov_urls.find_each(&:destroy!)
     searchgov_domain.destroy!
+    Resque::Job.destroy('searchgov',
```

Comment:
After helping with acceptance testing we realized that if we're currently indexing the domain, after  triggering the `SearchgovDomainDestroyerJob` , it stars deleting all the urls form the database but since the indexing hasn't been stopped yet, the SearchgovDomainIndexerJob is adding urls while the SearchgovDomainDestroyerJob is trying to delete them.

Can you please dequeue the job first thing then start deleting the urls?